pom.xml

```diff
@@ -200,8 +200,10 @@
 			        <doclet>com.iota.mdxdoclet.MDXDoclet</doclet>
 			        <sourcepath>src/main/java</sourcepath>
 			        <useStandardDocletOptions>false</useStandardDocletOptions>
+			        <!--
 			        <reportOutputDirectory>${project.basedir}</reportOutputDirectory>
           			<destDir>apidocs</destDir>
+          			-->
```

Comment:
Final nitpicking and then merge,
Instead of commenting out please erase these lines :-)

I know we have plenty of comments on the pom, but that doesn't make it right. I prefer to have a clean pom.

Reply:
Lets go!